src/client/opamClient.ml

```diff
@@ -637,6 +637,7 @@ let update
           (OpamPackage.Set.to_json updates);
       (success, not (OpamPackage.Set.is_empty updates)), st
   in
+  OpamSwitchState.drop _st;
```

Comment:
Please remove the leading `_` if actually using the variable, `_foo` should be reserved for documentation of unused variables.

Reply:
I kept the `_` because they are not really used, just kept to be dropped. Updated!